src/components/keypad.js

```diff
@@ -1,30 +1,35 @@
-      <button className="key" type="button">%</button>
-      <button className="key orange" type="button">/</button>
-      <button className="key" type="button">7</button>
-      <button className="key" type="button">8</button>
-      <button className="key" type="button">9</button>
-      <button className="key orange" type="button">x</button>
-      <button className="key" type="button">4</button>
-      <button className="key" type="button">5</button>
-      <button className="key" type="button">6</button>
-      <button className="key orange" type="button">-</button>
-      <button className="key" type="button">1</button>
-      <button className="key" type="button">2</button>
-      <button className="key" type="button">3</button>
-      <button className="key orange" type="button">+</button>
-      <button className="key zero" type="button">0</button>
-      <button className="key" type="button">.</button>
-      <button className="key orange" type="button">=</button>
+      <button className="key" type="button" onClick={() => onButtonClick('AC')} data="AC">AC</button>
+      <button className="key" type="button" onClick={() => onButtonClick('+/-')} data="+/-">+/-</button>
+      <button className="key" type="button" onClick={() => onButtonClick('%')} data="%">%</button>
+      <button className="key orange" type="button" onClick={() => onButtonClick('÷')} data="÷">÷</button>
+      <button className="key" type="button" onClick={() => onButtonClick('7')} data="7">7</button>
+      <button className="key" type="button" onClick={() => onButtonClick('8')} data="8">8</button>
+      <button className="key" type="button" onClick={() => onButtonClick('9')} data="9">9</button>
+      <button className="key orange" type="button" onClick={() => onButtonClick('x')} data="x">x</button>
+      <button className="key" type="button" onClick={() => onButtonClick('4')} data="4">4</button>
+      <button className="key" type="button" onClick={() => onButtonClick('5')} data="5">5</button>
+      <button className="key" type="button" onClick={() => onButtonClick('6')} data="6">6</button>
+      <button className="key orange" type="button" onClick={() => onButtonClick('-')} data="-">-</button>
+      <button className="key" type="button" onClick={() => onButtonClick('1')} data="1">1</button>
+      <button className="key" type="button" onClick={() => onButtonClick('2')} data="2">2</button>
+      <button className="key" type="button" onClick={() => onButtonClick('3')} data="3">3</button>
+      <button className="key orange" type="button" onClick={() => onButtonClick('+')} data="+">+</button>
+      <button className="key zero" type="button" onClick={() => onButtonClick('0')} data="0">0</button>
+      <button className="key" type="button" onClick={() => onButtonClick('.')} data=".">.</button>
+      <button className="key orange" type="button" onClick={() => onButtonClick('=')} data="=">=</button>
     </div>
   );
 }
 
```

Comment:
- [ ] `required`
- [ ] Amazing work so far 👏 
- [ ] however, I'm getting in the keypad.js  please see the error below on the screen and please fix it 🥂 

![image](https://github.com/redmarverick/Math-magicians/assets/66230306/7394b8ca-9a9e-4e36-8163-b5f276cccefa)